changelogs/fragments/470-spacewalk-legacy-python-certificate-validation.yaml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - rhn_channel.py (spacewalk) - Python 2.7.5 fails if the certificate should not be validated. Fixed this by creating the correct ssl_context. (https://github.com/ansible-collections/community.general/pull/470)
```

Comment:
```suggestion
  - rhn_channel - Python 2.7.5 fails if the certificate should not be validated. Fixed this by creating the correct ssl_context (https://github.com/ansible-collections/community.general/pull/470).
```

Reply:
any updates on this? https://docs.ansible.com/ansible/latest/community/development_process.html#changelogs-how-to